src/daidepp/keywords/press_keywords.py

```diff
@@ -210,6 +61,11 @@ class ALYVSS(_DAIDEObject):
     aly_powers: List[Power]
     vss_powers: List[Power]
 
+    def __init__(self, aly_power, vss_powers):
```

Comment:
Why is this necessary? Also, shouldn't `aly_power` be plural? (`aly_powers`)

Reply:
not necessary. Idk why this is in there